gnmi_server/client_subscribe.go

```diff
@@ -174,6 +188,8 @@ func (c *Client) Run(stream gnmipb.GNMI_SubscribeServer) (err error) {
 	c.Close()
 	// Wait until all child go routines exited
 	c.w.Wait()
+	// delete connectionkey under ip/connections
+	connectionManager.Remove(connectionKey)
```

Comment:
There are some other "return grpc.Errorf(...)" in this function, do we need to remove connection for them?

Reply:
Yes, good catch. Done